.travis.yml

```diff
@@ -94,7 +95,7 @@ matrix:
           astyle
       before_script:
       script:
-      - scripts/style-checker.sh check
+      - scripts/style-checker.sh diff
```

Comment:
Yes, that was what we have agreed upon some weeks ago, wasn't it? We could
do fine tuning of diff options, though.

It helped me fix the issues detected in this case for example.

On Tue, Sep 6, 2016 at 12:43 PM, Budai Laszlo <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/balabit/syslog-ng/pull/1194#discussion_r77611845>:
>
> > @@ -94,7 +95,7 @@ matrix:
> >            astyle
> >        before_script:
> >        script:
> > -      - scripts/style-checker.sh check
> > +      - scripts/style-checker.sh diff
>
> Do we really want to put the diff in the travis log?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/balabit/syslog-ng/pull/1194/files/13268b5522cb402e8d1464c73b2f9972e5d4cae6..9dad1cb2d4e02d302fbc9a7bffc772606179235d#r77611845>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AOobpJgKAQajHdZaOBvvl2tqy_AZLyolks5qnUPSgaJpZM4J0Jgy>
> .
>